MetamodelAlgemeen.md

```diff
@@ -853,7 +853,7 @@ In sommige gevallen kan de `URI` van een modelelement niet bepaald worden aan de
   <dfn>is gedefinieerd in</dfn>De package waarin het modelelement gedefinieerd is.
 </aside>
 
-*Toelichting* De definiërende _package_ is meestal de _package_ die het modelelement bevat. De waarde voor dit metagegeven kan wanneer dit het geval is afgeleid worden. In afwijkende situaties moet de `URI` van de betreffende _package_ ingevuld worden. Een _package_ van het type <code><a>View</a></code> definieert nooit de modelelementen die het bevat, dit is altijd een ander _package_ van het type <code><a>Domein</a></code>. Het verschil met het metagegeven <code><a>Herkomst</a></code> is dat dit een directe verwijzing is naar een informatiemodel of een _package_ daarbinnen. 
+*Toelichting* De definiërende _package_ is meestal de _package_ die het modelelement bevat. De waarde voor dit metagegeven kan, wanneer dit het geval is, afgeleid worden. In afwijkende situaties moet de `URI` van de betreffende _package_ ingevuld worden. Een _package_ van het type <code><a>View</a></code> definieert nooit de modelelementen die het bevat, dit is altijd een ander _package_ van het type <code><a>Domein</a></code>. Het verschil met het metagegeven <code><a>Herkomst</a></code> is dat `Is gedefinieerd in` een directe verwijzing is naar een informatiemodel of een _package_ daarbinnen door middel van een `URI`. 
```

Comment:
Ik zet de reviewopmerking even hierbij, maar het slaat eigenlijk op iets dat er al stond (bij URI en basis-URI). Volgens mij is dat niet helemaal correct. Het voorbeeld <urn:object:Pand> zul je eigenlijk ooit kunnen afleiden, want je zult geen package met de basis-URI "urn:object" hebben (!). Mijn voorstel zou zijn om, indien er geen basis-URI's zijn ingevuld, de volgende opbouw te hanteren: "urn:model:<package-tree>:<gekwalificeerde-naam>, waarbij de gekwalificeerde naam de naam is van het modelelement, behalve bij attribuuttypen enzo, waarbij ook nog het objecttype meekomt. Voorbeeld: <urn:model:imbag:kern:nummeraanduiding.huisnummer>. Wellicht is daarbij "modelitem" beter dan "model" overigens (NB: het ding achter urn zou een kwalificatie moeten zijn, dus urn:imbag is eigenlijk niet in lijn met de richtlijnen voor de opbouw van een URN

Reply:
De defaultwaarde moet zorgen dat ieder modelelement een unieke urn krijgt; want we kunnen niet uitsluiten dat twee modelelementen met dezelfde naam verschillend zijn. Dit geldt denk ik ook over packages heen. Is het per definitie fout om voor een package een naam te gebruiken die ook als naam voor een objecttype gebruikt wordt?  Denk aan een package "Locatie" waar objecttype "Locatie" in voor komt. Daarnaast zijn URNs over het algemeen case insensitive; Daarom is de prefix voor huisnummer ook belangrijk. 

Mijn voorstel is dan om de urn als volgt op te bouwen:
`urn:` + `modelelement` + `{informatiemodel.naam}` + {package.naam}` + `{modelelement.naam}`
Waarbij voor eigenschappen modelelement.naam volgt na de de naam van de eigenaar (bijvoorbeeld een objecttype).
Voor packages (wat ook modelelementen zijn) is het net anders.

- het gehele informatiemodel krijgt `urn:` + `modelelement` + `{informatiemodel.naam}`. bijvoorbeeld "urn:modelelement:geo"
- een domein package Locatie daarbinnen krijgt `urn:` + `modelelement` + `{informatiemodel.naam}` + {package.naam}`. Bijvoorbeeld "urn:modelelement:geo:locatie".
- Een objecttype Adres daarbinnen krijgt `urn:` + `modelelement` + `{informatiemodel.naam}` + {package.naam}` + `{modelelement.naam}`. Bijvoorbeeld "urn:modelelement:geo:locatie:adres"
- Een attribuutsoort volledigAdres bij locatie krijgt: `urn:` + `modelelement` + `{informatiemodel.naam}` + {package.naam}` + `{modelelement.naam}`. Bijvoorbeeld: urn:modelelement:geo:locatie:adres.volledigadres

De urn voor huisnummer is dan binnen de LV bijvoorbeeld: "urn:modelelement:imbaglv:objecten:nummeraanduiding.huisnummer